pkg/pwgen/pwgen.go

```diff
@@ -41,12 +43,18 @@ func GeneratePassword(length int, symbols bool) string {
 // GeneratePasswordCharset generates a random password from a given
 // set of characters
 func GeneratePasswordCharset(length int, chars string) string {
-	pw := &bytes.Buffer{}
-	for pw.Len() < length {
-		_ = pw.WriteByte(chars[randomInteger(len(chars))])
+	var password string
+
+	validator := crunchy.NewValidator()
+	for i := 0; i < 3 && validator.Check(password) != nil; i++ {
```

Comment:
You rely on the fact that the validator complains if the inital password is empty?

Reply:
Correct. Could be done "cleaner" with a couple more lines of code, but then I'm not sure it's actually more readable. What do you think?

Reply:
I think it's fine. Just wanted to understand your reasoning.